apps/experiments/models.py

```diff
@@ -231,6 +231,9 @@ class Experiment(BaseTeamModel):
     llm_provider = models.ForeignKey(
         "llm_providers.LlmProvider", on_delete=models.SET_NULL, null=True, blank=True, verbose_name="LLM Provider"
     )
+    llm_provider_new = models.ForeignKey(
```

Comment:
The plan is to remove the existing `llm_provider` app and then rename `llm_provider_new` to `llm_provider` here?